lib/compile.js

```diff
@@ -3,6 +3,7 @@
 const _ = require('lodash');
 const BbPromise = require('bluebird');
 const webpack = require('webpack');
+const tty = require('tty');
```

Comment:
The `tty` package is not part of the project's devDependencies. Can you add it to the package.json?

Reply:
The `tty` package is actually a node built-in. The npm package is a security holder now.

Reply:
Resolving since this is fine 👍